odoo/tools/pdf.py

```diff
@@ -19,6 +20,7 @@
 
 _logger = getLogger(__name__)
 DEFAULT_PDF_DATETIME_FORMAT = "D:%Y%m%d%H%M%S+00'00'"
+REGEX_SUBTYPE = re.compile('^\/[-\w+\.]+#2F[-\w+\.]+$')
```

Comment:
that's confusing on multiple aspects ^^

- please use raw strings for regex pattern declarations (the doc[1] explains why)
- did you actually meant to include the `+` character literally inside the character sets? Do you have an example where a subtype relevant for PDF/A includes a `+`?
- you don't need to escape the `.` inside a character set, it's taken literally.

[1] https://docs.python.org/3/library/re.html

Reply:
Thank you for the feedbacks, I've made the requested changes!